calico-enterprise/networking/configuring/qos-controls.mdx

```diff
@@ -130,3 +134,20 @@ spec:
 |---------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `qos.projectcalico.org/dscp` | Specifies the DSCP value of egress traffic toward an endpoint outside cluser or another host in cluster.<br/>Min value: `0`<br/>Max value: `63`<br/>String values: DF, EF, AF11, AF12, AF13, AF21, AF22, AF23, AF31, AF32, AF33, AF41, AF42, AF43, CS0, CS1, CS2, CS3, CS4, CS5, CS6                                                                                                                                     |
 
+## Troubleshooting
+
+### Bandwidth is not enforced on eBPF
+
+If limits for bandwidth on installations that use the eBPF data plane don't seem to be enforced, check calico-node logs to see if `tcx` may not be supported by
+the operating system with `kubectl logs`. For example:
```

Comment:
Suggested rephrasing for clarity:

If you use the eBPF data plane and bandwidth limits aren't being enforced, check the calico-node logs to see if the `tcx` module is supported by the operating system.
This module, which is present in kernel version 6.6 and later, is required for bandwidth limits to work with the eBPF data plane.

The following example shows logs that indicate an unsupported kernel version: